src/components/CheckboxInput/CheckboxInput.module.css

```
@@ -5,69 +5,67 @@
 \*------------------------------------*/
 
 /**
- * Wrapper for the checkbox, including the visually hidden input element.
- * 1) Setting position is required to place the absolutely positioned input element in the same spot as the SVG 
- *    when a (n^c)parent element in the tree is relatively positioned.
- * 2) line-height is set to be same as checkbox size to equal checkbox size and prevent browser styling.
+ * Wraps the visually hidden checkbox input element and the visible sibling svg element.
+ * 1) Centers the checkbox icon in the wrapper.
+ * 2) Aligns the checkbox with the first line of the label.
  */
 .input__wrapper {
-  position: relative; /* 1 */
-  line-height: var(--checkbox-svg-size); /* 2 */
-}
+  position: relative;
+
+  display: flex; /* 1 */
+  align-items: center;
 
+  align-self: flex-start; /* 2 */
+}
 /**
-   * The visually hidden input element for the checkbox. The visual checkmark is provided by an svg element.
-   * 1) Removes default margins placed by browser for checkboxes.
-   * 2) Remove the checkbox from the page flow, positioning it on top of the SVG.
-   * 3) Set same dimensions as the CheckboxSvg element.
-   * 4) Hide the input element visually.
-   *    This ensures the checkbox is still "physically" present so that all users,
-   *    especially on touch screen readers, still interact with the real checkbox element
-   *    where it would naturally be present.
-   */
+ * The visually hidden input element for the checkbox. The visual checkmark is provided by an svg element.
+ * 1) Removes default margins placed by browser for checkboxes.
+ * 2) Remove the checkbox from the page flow, positioning it on top of the SVG.
+ * 3) Hide the input element visually.
+ *    This ensures the checkbox is still "physically" present so that all users,
+ *    especially on touch screen readers, still interact with the real checkbox element
+ *    where it would naturally be present.
```

#3 and 4 have the same text. Is this supposed to be
```suggestion
 * 3) Set same dimensions as the CheckboxSvg element.
```

---

Yesssss thanks for the catch 👍